### ISSUE TITLE
react-native-maps@0.20.0 requires a peer of react-native@0.51.0 but none was installed.

### ISSUE THREAD

**Issue by JeromeDesseaux** [react-community/react-native-maps#2023]
Hi everyone and thanks for everything you've mafe here! It look really awesome. 

I'm starting React Native and I'm facing a NPM dependency error. 
`npm WARN react-native-maps@0.19.0 requires a peer of react@16.0.0 but none was installed.
npm WARN react-native-maps@0.19.0 requires a peer of react-native@0.51.0 but none was installed.
npm WARN react-native-maps@0.20.0 requires a peer of react@16.0.0 but none was installed.
npm WARN react-native-maps@0.20.0 requires a peer of react-native@0.51.0 but none was installed.
`

## config 

`{
  "name": "AwesomeProject",
  "version": "0.1.0",
  "private": true,
  "devDependencies": {
    "react-native-scripts": "1.11.1",
    "jest-expo": "25.0.0",
    "react-test-renderer": "16.2.0"
  },
  "main": "./node_modules/react-native-scripts/build/bin/crna-entry.js",
  "scripts": {
    "start": "react-native-scripts start",
    "eject": "react-native-scripts eject",
    "android": "react-native-scripts android",
    "ios": "react-native-scripts ios",
    "test": "node node_modules/jest/bin/jest.js"
  },
  "jest": {
    "preset": "jest-expo"
  },
  "dependencies": {
    "expo": "^25.0.0",
    "react": "^16.0.0",
    "react-native": "^0.52.0",
    "react-native-maps": "^0.20.0"
  }
}
`

node 8.9.4 / npm 4.6.1

## Steps to reproduce 

I really dont know. Its a fresh install of both operating system and nodejs/react. I just added the dependency to RN-maps. Nothing more.

## What did I tried

 If I try rolling back RN and R to previous versions as required by the error message. I got and "javascript version mismatch" error from Expo. 

Thanks for reading me :)

**Comment by rborn** [comment]
@JeromeDesseaux is this breaking your app or it's just a nuisance in the logs?

**Comment by mellet** [comment]
Not 100% sure.
But if your in Expo don't they ship their own version that works in Expo?

Meaning you should `import { MapView } from "expo"`

**Comment by JeromeDesseaux** [comment]
I followed the React-Native's starting guide [here](https://facebook.github.io/react-native/docs/getting-started.html)

But apparently, the required way (RN-maps) is using react-native init. Using this command, I got ios and android folders and, RN-maps is working. I still got theses warning. But, at least, it works. 

I'm hoping this would help someone (a beginner as I am maybe) : Scaffold your project using react-native command, not the one provided by this guide. 

Thanks for your help anyway :)
